diff --git a/dist/runtime/server/navigation-query.js b/dist/runtime/server/navigation-query.js
index e7ed33379a63dc3d4306452c42ad4136a0243e0b..72242293b6ed72c3b31723e9b1bef54889253f6f 100644
--- a/dist/runtime/server/navigation-query.js
+++ b/dist/runtime/server/navigation-query.js
@@ -61,7 +61,7 @@ export async function resolveContentNavigation(event, query, navigationOptions =
         localeFilter,
         draftFilter
       ));
-      const configsPlan = trustedNavigationPlan(query.plan, query.collection, andFilters(
+      const configsPlan = trustedNavigationPlan(query.plan, null, andFilters(
         { type: "compare", field: "navigationFile", operator: "eq", value: true },
         { type: "compare", field: "partial", operator: "eq", value: true },
         localeFilter,
